proto/plan_common.proto

```diff
@@ -190,3 +184,31 @@ message Cardinality {
 message ExprContext {
   string time_zone = 1;
 }
+
+message AdditionalColumnNormal {}
+
+message AdditionalColumnKey {}
+
+message AdditionalColumnTimestamp {}
+
+message AdditionalColumnPartition {}
+
+message AdditionalColumnOffset {}
+
+message AdditionalColumnFilename {}
+
+message AdditionalColumnHeader {
+  optional string inner_field = 1;
+}
```

Comment:
I think we had better consider the `include header <key>` and `include header` as 2 separated cases because they are quite different.

To differentiate, we may call the later one `AdditionalColumnHeaders` (adding an `s`).

Reply:
fixed